### PR TITLE
Upgrade ReactiveCocoa from 2.3.1 -> 2.4.7, and bump version from 0.5.0 to 0.5.1

### PR DESCRIPTION
**Pull request by inlinestyle**
review @confucious

### CODE REVIEW

File: GCMFormTableView.podspec [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
   s.author       = { "Jerry Hsu" => "jerry.hsu@gamechanger.io" }
-  s.source       = { :git => "https://github.com/gamechanger/GCMFormTableView.git", :tag => "0.5.0" }
+  s.source       = { :git => "https://github.com/gamechanger/GCMFormTableView.git", :tag => "0.5.1" }
   s.source_files = "GCMFormTableView/GCMFormTableView/*.{h,m}"
   s.license = { :type => 'MIT', :type => 'LICENSE' }
   s.requires_arc = true
   s.ios.deployment_target = '6.0'
   s.dependency 'GCMCocoaExtensions', '~> 0.2.0'
-  s.dependency 'ReactiveCocoa', '~> 2.3.1'
+  s.dependency 'ReactiveCocoa', '~> 2.4.7'
```
Comment (confucious) [inline GCMFormTableView.podspec]: The trick with these projects is you need to edit `script/GCMFormTableView.podspec.erb` to update the dependencies. the version number of this project itself gets handled by the publish script.

Comment (inlinestyle) [inline GCMFormTableView.podspec]: So do I not commit changes to this file? Or do I just also need the change reflected in the `.erb` file?

Comment (confucious) [inline GCMFormTableView.podspec]: Not sure, but should be fine to leave it in just in case.

**Comment by confucious** [comment]
:+1:

**Pull request merged**